doc/Makefile

```diff
@@ -5,10 +5,10 @@ include Makefile.sphinx
 
 CHPL2RST     = ../util/run-in-venv.bash ./util/chpl2rst.py
 
-CHPL_DOC_PRIMERS_DIR = primers/primers
+CHPL_DOC_PRIMERS_DIR = $(SOURCEDIR)/primers/primers
```

Comment:
I realize this is pre-existing, but it's always surprised me when I cut and paste a URL with primers/primers/ in it.  I think I used to know why we had this, but every time I've run into it since it's seemed surprising...